platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/startup/CommandLineArgs.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.startup;
+
+import com.swirlds.common.system.NodeId;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Command line arguments that can be passed to the broswer when starting up
+ * @param localNodesToStart the set of nodes to start on this machine
+ */
+public record CommandLineArgs(@NonNull Set<NodeId> localNodesToStart) {
+    /**
+     * Parse the command line arguments passed to the main method
+     * @param args the arguments to parse
+     * @return the parsed arguments
+     */
+    public static CommandLineArgs parse(@Nullable final String[] args) {
```

Comment:
should it really be nullable? passing an empty array is fine, but null?

Reply:
changed